manifest.json

```diff
@@ -42,14 +46,15 @@
     },
     "content_scripts": [
       {
-        "matches": ["https://*.duckduckgo.com/*", "https://duckduckgo.com/*"],
+        "matches": ["<all_urls>"],
+        "all_frames": true,
         "css": ["css/noatb.css"]
       }
     ],
     "permissions": [
         "contextMenus",
         "webRequest",
         "webRequestBlocking",
-        "*://*.duckduckgo.com/"
+        "*://*/*"
```

Comment:
@MariagraziaAlastra @jdorweiler just out of curiosity, will this show that nasty line on install, saying that this extension will be able to read all your data on all sites you visit? Last time I tried it did, so I guess it may be something to consider putting into the description.